be1-go/internal/database/sqlite/lao.go

```diff
@@ -7,7 +7,7 @@ import (
 	"errors"
 	"go.dedis.ch/kyber/v3"
 	poperrors "popstellar/internal/errors"
-	"popstellar/internal/handler/channel"
+	message2 "popstellar/internal/handler/message"
```

Comment:
Same as the previous remark regarding the hub module import.

Reply:
fixed